src/converters/cover.ts

```diff
@@ -27,12 +27,28 @@ export class CoverCreator implements ServiceCreator {
           exposesHasFeatures(e) &&
           !accessory.isServiceHandlerIdKnown(CoverHandler.generateIdentifier(e.endpoint))
       )
-      .forEach((e) => this.createService(e as ExposesEntryWithFeatures, accessory));
+      .forEach((e) => {
+        let exposesMotorState: boolean = false;
+        const motorStateExpose = exposes.find(
+          (m) =>
+            CoverHandler.generateIdentifier(m.endpoint) === CoverHandler.generateIdentifier(e.endpoint) &&
+            m.type === ExposesKnownTypes.ENUM &&
+            m.name === 'motor_state' &&
+            exposesHasEnumProperty(m) &&
```

Comment:
Should also check if it is published by Zigbee2MQTT.

```suggestion
            exposesIsPublished(m) &&
            exposesHasEnumProperty(m) &&
```